Add the 'linked' style to the preferences page chooser for the benefit
of (among others) the slim-nimbus theme, so the buttons render in a
linked way, as in gnome-system-monitor.

--- epiphany-40.0/src/resources/gtk/prefs-dialog.ui.~1~	2021-03-19 19:43:48.000000000 +0000
+++ epiphany-40.0/src/resources/gtk/prefs-dialog.ui	2021-03-27 16:52:03.278895515 +0000
@@ -10,6 +10,9 @@
     <property name="default-width">640</property>
     <property name="default-height">800</property>
     <property name="can-swipe-back">True</property>
+    <style>
+      <class name="linked"/>
+    </style>
     <signal name="key-press-event" handler="on_key_press_event"/>
     <signal name="delete-event" handler="on_delete_event"/>
     <child>
